power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/math_solver.hpp

```diff
@@ -171,6 +173,9 @@ template <bool sym> class MathSolver {
 
 template class MathSolver<true>;
 template class MathSolver<false>;
+} // namespace math_solver
+
+using math_solver::MathSolver;
 
 } // namespace power_grid_model
```

Comment:
for this PR not really an issue but usually, it's good practice to not clog the parent namespace except for very rare cases. The `MathSolver` is an edge case in which it doesn't matter, but the individual solvers should be restricted to the `math_solver` namespace, only